src/main/java/org/jabref/logic/layout/format/HTMLChars.java

```diff
@@ -23,6 +23,15 @@ public class HTMLChars implements LayoutFormatter {
      * */
     private static final Pattern HTML_ENTITY_PATTERN = Pattern.compile("&(?!(?:[a-z0-9]+|#[0-9]{1,6}|#x[0-9a-fA-F]{1,6});)");
 
+    private boolean keep = false;
+
+    @Override
+    public void setArgument(String arg) {
+        if ("keep".equals(arg)) {
+            this.keep = true;
+        }
```

Comment:
I would also use the full name `keepCurlyBraces` here, because otherwise, a reader does not understand what `keep` means. Future work could keep latex commands or something else.

```suggestion
        this.keep = "keepCurlyBraces".equalsIgnoreCase(arg);
```